maven/lib/dependabot/maven/new_version.rb

```diff
@@ -0,0 +1,68 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "dependabot/maven/version_parser"
+require "dependabot/version"
+require "dependabot/utils"
+
+# See https://maven.apache.org/pom.html#Version_Order_Specification for details.
+
+module Dependabot
+  module Maven
+    class NewVersion
```

Comment:
I'm assuming the `NewVersion` name is temporary pending the swap of the actual `Version` class?

Reply:
That is correct. My next step is renaming this to version.rb and updating the tests etc.